index.bs

```diff
@@ -3253,14 +3253,35 @@ extended attribute’s [=takes a named argument list|named argument list=].
 </div>
 
 Two types are <dfn id="dfn-distinguishable" export>distinguishable</dfn> if
-at most one of the two [=includes a nullable type=], is a [=dictionary type=], or a [=record type=],
-and at least one of the following three conditions is true:
+the following algorithm returns <i>true</i>.
 
-1.  The two types (taking their [=inner types=]
+<ol class="algorithm">
+1.  If both types are either a [=union type=] or nullable union type,
+    return <i>true</i> if each member type of the one
```

Comment:
Yeah, I wasn't sure what to do with these. They're a little like the return codes in https://w3c.github.io/webappsec-secure-contexts/#settings-object, but `<code>` didn't really seem right. They're also not the JS **true** and **false**, so `<emu-val>` wasn't right. I can leave them as plain text if you prefer.